iphone/Classes/TiUIAlertDialogProxy.m

```diff
@@ -131,94 +114,73 @@ -(void)show:(id)args
 
         style = [TiUtils intValue:[self valueForKey:@"style"] def:UIAlertViewStyleDefault];
         
-        if ([TiUtils isIOS8OrGreater]) {
-            RELEASE_TO_NIL(alertController);
-            [[[TiApp app] controller] incrementActiveAlertControllerCount];
-            
-            alertController = [[UIAlertController alertControllerWithTitle:[TiUtils stringValue:[self valueForKey:@"title"]]
-                                                                  message:[TiUtils stringValue:[self valueForKey:@"message"]]
-                                                            preferredStyle:UIAlertControllerStyleAlert] retain];
-            int curIndex = 0;
-            
-            //Configure the Buttons
-            for (id btn in buttonNames) {
-                NSString* btnName = [TiUtils stringValue:btn];
-                if (!IS_NULL_OR_NIL(btnName)) {
-                    
-                    UIAlertActionStyle alertActionStyle;
-                    
-                    if (curIndex == cancelIndex) {
-                        alertActionStyle = UIAlertActionStyleCancel;
-                    } else if (curIndex == destructiveIndex) {
-                        alertActionStyle = UIAlertActionStyleDestructive;
-                    } else {
-                        alertActionStyle  = UIAlertActionStyleDefault;
-                    }
-
-                    UIAlertAction* theAction = [UIAlertAction actionWithTitle:btnName
-                                                                        style:alertActionStyle
-                                                                      handler:^(UIAlertAction * action){
-                                                                                [self fireClickEventWithAction:action];
-                                                                                }];
-                    [alertController addAction:theAction];
+        RELEASE_TO_NIL(alertController);
```

Comment:
Sorry, it doesn't. The indentation just changed because of the removed if-statement. Continuing review ...